acpi_tables/src/aml.rs

```diff
@@ -262,6 +263,38 @@ impl Aml for EisaName {
     }
 }
 
+// UUID (Universally Unique IDentifiers)
+pub struct Uuid {
+    pub data: Vec<u8>,
+}
+
+// Converts an ASCII UUID or GUID string to an encoded 128-bit Buffer object.
+// The UUID string is in the format of aabbccdd-eeff-gghh-iijj-kkllmmnnoopp,
+// where aa – pp are one byte hexadecimal numbers, made up of hexadecimal digits.
+impl Uuid {
+    pub fn new(str: String) -> Self {
```

Comment:
This isn't ACPI specific. This is just a standard UUID definition. So this doesn't belong in this crate. Use the `uuid` crate in vmm to parse the string.

Reply:
Aha, yes, I removed it. The endian of the UUID is a little bit specific (mixed-endian) but still standard (https://en.wikipedia.org/wiki/Universally_unique_identifier#Encoding).  We can use `uuid` crate.